### PR TITLE
doc-house-keeping 

### PR DESCRIPTION
**Pull request by mmiele**
- [x] Move links from HLD document to the main README file
- [x] Add FAQ link
- [x] Fix SAI link
- [x] Fix HLD deployment diagram

### CODE REVIEW

File: documentation/README.md [hunk 1]
```diff
@@ -65,4 +65,27 @@ Some of the guiding principles for this aproach are:
...
+- [IPDK](https://ipdk.io/) and [IPDK GitHub](https://github.com/ipdk-io/ipdk-io.github.io)
+- [bmv2 - behavioral model v2](https://github.com/p4lang/behavioral-model)
+- [DPDK](https://www.dpdk.org)
+
+
+## References
+
+- [Glossary](https://github.com/Azure/DASH/wiki/Glossary)
+- [FAQ](https://github.com/Azure/DASH/wiki/FAQ)
+- [SAI](./sai/README.md)
```
Comment (chrispsommers) [inline documentation/README.md:86]: Please remove everything from here down - they're internal links which are already stale and redundant with other ToCs we've created. Thx.

**Review by chrispsommers — CHANGES_REQUESTED** [review]

**Review by KrisNey-MSFT — APPROVED** [review]

File: documentation/README.md (hunk 1 above)
Comment (mmiele) [reply documentation/README.md:86]: Removed the links.

Comment (chrispsommers) [reply documentation/README.md:86]: Links are still visible???

**Review by chrispsommers — CHANGES_REQUESTED** [review]

File: documentation/README.md (hunk 1 above)
Comment (mmiele) [reply documentation/README.md:86]: It helps if I commit the changes. 🌞 
Done.

**Review by KrisNey-MSFT — APPROVED** [review]
Great!

**Pull request merged**